notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,11 +18,11 @@
   },
```

Comment:
<p>I'm pretty sure there should only be one value for duration here, not an entire list. I think the entire list is getting duplicated throughout all the entries for the same test.  Try something like the following to distribute the right duration to the right test entry</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp;decoded = [list(zip(timestamps, g, duration)) for g in decoded]

</pre><p> </p><p>The duration, should be the same size and treated similar to the timestamp list, and zip'ed along with it as well </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
@MichaelClifford We have fixed the the one value for duration issue in the latest commit.

As a note: we are only looking at time durations for the Test names that are "Overall". For the other Tests, we store time_duration as None. This is because not all tests capture the graph metric values and also the tests which capture the values do not capture it for all time durations, so we thought it made sense to only capture it for the "Overall" tests as that is also what we need for the KPI calculations.